homeassistant/components/history.py

```diff
@@ -118,6 +118,30 @@ def state_changes_during_period(hass, start_time, end_time=None,
     return states_to_json(hass, states, start_time, entity_ids)
 
 
+def get_last_state_changes(hass, number_of_states=10, entity_id=None):
```

Comment:
we should not have a default param for `number_of_states`. I'm not even sure if we should allow an empty entity_id because it makes little sense. Let's just start with no defaults for either.

Reply:
Fun fact: we used to have this method and it was shown in the frontend but once we got graphs it got unused and I removed it.